### PR TITLE
Adjust docs formatting for incorporation in MRW doc submodule

### PR DESCRIPTION
**Pull request by gspetro-NOAA**
This change affects only formatting (no content changes). However, the changes will allow us to include the spack-stack documentation within a single MRW App spack-stack installation chapter. Without this change, we get each spack-stack docs chapter separately: 
image
With this change, the result will be a single spack-stack chapter inserted in the MRW documentation after Chapter 4.

### CODE REVIEW

**Review by climbfuji — COMMENTED** [review]
@gspetro-NOAA Thanks for these updates. I am fine with the changes, the standalone build of the documentation looks as before. Is it ok to change the target branch for this PR to the release/v1 branch?

**Comment by climbfuji** [comment]
I changed the base to release/v1 - will approve and merge now.

**Review by climbfuji — APPROVED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -3,7 +3,7 @@
   #url = https://github.com/spack/spack
   #branch = develop
   url = https://github.com/NOAA-EMC/spack
-  branch = release/jcsda_emc_release_v1
+  branch = jcsda_emc_spack_stack
```
Comment (climbfuji) [inline .gitmodules]:
```suggestion
  branch = release/jcsda_emc_release_v1
```

**Review by climbfuji — CHANGES_REQUESTED** [review]
Need to update `.gitmodules`

**Review by climbfuji — APPROVED** [review]

**Comment by gspetro-NOAA** [comment]
@climbfuji Looks like you took care of any concerns already. :) Anything else you'd like me to do?

**Comment by climbfuji** [comment]
No, just need to wait for the CI tests to pass and then we are good to go.

**Pull request merged**